orchestration/values.yaml

```diff
@@ -0,0 +1,17 @@
+volumes:
+  storageClass: standard
+  downloadSize: 1Gi
+  extractSize: 10Gi
+cron:
+  enable: true
+  # Every Sunday at 3:30am
+  schedule: '30 3 * * Sun'
+  timezone: 'America/New_York'
+dataflow:
+  region: us-west-1
```

Comment:
I updated the region, but the rest of these values were copied from https://github.com/DataBiosphere/clinvar-ingest/blob/master/orchestration/values.yaml

Reply:
We can go more lightweight for ENCODE, I think. Let's start with `n1-standard-1` and see if we need to bump things up.